lib/reconcile/astarte_generic_backend.go

```diff
@@ -170,6 +170,9 @@ func getAstarteGenericBackendEnvVars(deploymentName string, cr *apiv1alpha1.Asta
 		Value: getCassandraNodes(cr),
 	})
 
+	// Add Cassandra connection options
+	ret = appendCassandraConnectionEnvVars(ret, "CASSANDRA", cr)
```

Comment:
Why are you explicitly setting a prefix here? Has anything changed in our vars? Remember that RabbitMQ has that requirement due to retrocompatibility issues.